.github/workflows/deploy-to-gh-pages.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches: [ master ]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: master
+        path: website
+      
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10'
+
+    - name: Install Gulp CLI
+      run: npm install -g gulp-cli
+
+    - name: Inject Localize token
+      run: sed -i -e 's/localize-key/vWKzIgeZa4D3c/g' website/src/handlebars/partials/header.handlebars
+
+    - name: Build
+      run: |
+        cd website
+        npm install
+        gulp build
+
+    - name: Create Directory Stucture
+      run: |
+        mkdir tmp
+        mv website/sitemap.xml tmp/
+        cd website
+        find . -type f ! -name '*.html' ! -name '*.pdf' -maxdepth 1 -mindepth 1  -delete
+        find . -type d -not -name 'dist' -not -name '.git' -maxdepth 1 -mindepth 1 -exec rm -rf {} \;
```

Comment:
What's the purpose of those two?

Reply:
in order to deploy to github pages we need to remove the development files. Gulp doesn't do this because it would be impossible to develop at the same time.

I'm happy to look at optimisations in the future but for now I'm just using the scripts from the jenkins job